app/views/admin/users/show.html.haml

```diff
@@ -7,12 +7,16 @@
     %p
       Account active since:
       = @user.created_at.to_s(:short)
+  .govuk-grid-column-one-third
+    %h2.govuk-heading-s
+      Actions
+    %p
+      = link_to 'Link to a supplier', new_admin_user_membership_path(@user)
 
-= link_to 'Link to a supplier', new_admin_user_membership_path(@user)
-
-.govuk-grid-row
+.govuk-grid-row{:class => 'govuk-!-margin-top-7'}
```

Comment:
I much prefer this layout spacings on the row div like this rather than on an element like the table below, I think we should stick to this approach, be consistent and add more rows to the grid.